src/main/java/core/basesyntax/model/FruitType.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public enum FruitType {
+    BANANA, APPLE
+}

```

Comment:
Probably not a mistake, but not very flexible approach, you need to amend and recompile your program to be able to accept new fruit types